packages/common/src/lib/format-applicant.js

```diff
@@ -1,19 +1,63 @@
+const escape = require('escape-html');
+
 /**
  * @typedef {import("appeals-service-api").Api.ServiceUser} ServiceUser
+ * @typedef {import('@pins/common/src/constants').AppealToUserRoles} AppealToUserRoles
+ * @typedef {import('@pins/common/src/constants').LpaUserRole} LpaUserRole
+ */
+
+/**
+ * @typedef {Object} formattedApplicant
+ * @property {object} key
+ * @property {object} value
  */
 
-const { APPEAL_USER_ROLES } = require('../constants');
+const { APPEAL_USER_ROLES, LPA_USER_ROLE } = require('../constants');
 
 /**
  * @param {ServiceUser[]} [users]
- * @returns {string}
+ * @param {AppealToUserRoles|LpaUserRole|null} userType
+ * @returns {formattedApplicant}
  */
-const formatApplicant = (users) => {
-	const appellant = users?.find((x) => x.serviceUserType === APPEAL_USER_ROLES.APPELLANT);
+const formatApplicant = (users, userType = APPEAL_USER_ROLES.INTERESTED_PARTY) => {
+	const appellant = users?.find((user) => user.serviceUserType === APPEAL_USER_ROLES.APPELLANT);
+
+	if (userType === LPA_USER_ROLE) {
+		const agent = users?.find((user) => user.serviceUserType === APPEAL_USER_ROLES.AGENT);
+		if (agent) {
+			return {
+				key: { text: 'Agent contact details' },
+				value: { html: formatContactDetails(agent) }
+			};
+		}
+		if (appellant) {
+			return {
+				key: { text: 'Appellant contact details' },
+				value: { html: formatContactDetails(appellant) }
+			};
+		}
+	}
 
-	if (!appellant) return '';
+	if (appellant)
+		return {
+			key: { text: 'Applicant' },
+			value: { text: `${appellant.firstName} ${appellant.lastName}` }
```

Comment:
sorry think we we need it here as well

Reply:
even if the value is set as text rather than html?

Reply:
good point